spendingPlansConfig.example.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "id": "c758c095-342c-4607-9db5-867d7e90ab9d",
```

Comment:
CC: @Nana-EC @ebadiere 

Well, there are two approaches in which this could be handled:
1. Clear all spending plans from the cache on every start-up and re-populate them from the config file:
   - in this case we don't need to specify IDs in the config file, they can be randomly generated on each startup  
   - historical logs on mainnet and testnet will be harder to debug because we would have generated new IDs on redeploys / restarted pods and the IDs we have from the logs before this moment wouldn't be valid anymore
   - if Kubernetes pods are restarted - the cache would be re-populated with the same spending plans but with new IDs and we would lose track of the current spending as it will also be restarted
2. Only populate the "delta" from the config file (i.e., delete "obsolete" spending plans which are no longer present in the config file and create new spending plans which were not specified before):
   - in this case we need to have an ID specified in the config file, otherwise we cannot know if a given spending plan was already populated or not in the cache
   - historical logs on mainnet and testnet will be easy to debug since the same ID we are seeing in the logs, will be still valid and we can retrieve the spending plan from the cache by that ID
   - the Kubernetes pods can be freely restarted without affecting the behavior of the hbar limiter, as old spending plans wouldn't be cleared and only new ones that have appeared in the config file will be populated

Reply:
> How would someone unfamiliar with the codebase, or loading the config file for the first time, come up with such IDs

You don't need to be familiar with the codebase to come up with a random UUID: https://www.uuidgenerator.net/version4

> I was under the impression that these IDs are auto-generated by the Relay and only used internally.

They are if an `"id"` field is not specified in the config file.

Reply:
If I understand this correctly, with option 2, the config file can be the single source of truth regarding the spending plan.

Reply:
discussed off line and agreed on option 2 mentioned above. However, needed to come up with a plan to construct the plan ID beforehand and make it as a required field.